pymatgen/command_line/tests/test_bader_caller.py

```diff
@@ -78,6 +78,8 @@ def test_from_path(self):
         assert np.allclose(charge, charge0)
         if os.path.exists("CHGREF"):
             os.remove("CHGREF")
+        if os.path.exists(os.path.join(test_dir, "CHGREF")):
+            os.remove(os.path.join(test_dir, "CHGREF"))
```

Comment:
Would probably be best to use the `PymatgenTest.pmt_path` fixture so we don't have to worry about manually cleaning up test-generated files.

Reply:
Is `PymatgenTest.tmp_path` already implemented? If yes, I need some time to look into how to refactor this part. If no, I need more time 😂

Reply:
Yes, right here

https://github.com/materialsproject/pymatgen/blob/71b7aec2b6199b8f5b00464ee98e6a30cee441e1/pymatgen/util/testing.py#L50-L54

Reply:
Thanks I am looking into it!